CONTRIBUTORS.md

```diff
@@ -224,6 +225,7 @@
  - [Detector1](https://github.com/Detector1)
  - [BlackIce013](https://github.com/blackice013)
  - [mporcas](https://github.com/mporcas)
+ - [TelepathicWalrus](https://github.com/TelepathicWalrus)
```

Comment:
This is the section for contributors to the Emby source code from before it was forked by Jellyfin. You should only be adding your name to the Jellyfin section.
```suggestion
```